ivy/container/sorting.py

```diff
@@ -48,6 +127,51 @@ def argsort(
         *,
         out: Optional[ivy.Container] = None
     ) -> ivy.Container:
+        """
+        ivy.Container instance method variant of ivy.argsort. This method simply wraps the
+        function, and so the docstring for ivy.argsort also applies to this method
+        with minimal changes.
+        
+        Parameters
+        ----------
+        self 
+            input array.
```

Comment:
`self` should always be an `ivy.Container` for a container instance method, please update this line stating `self` as an input container.